Fix mocha modal

usirin opened the pull request:
anonrig/koding:master needs to be rebased against koding/koding:master, that's why i didn't want to force push to your master.

Instead of manually updating the options object, ReactView instances now accepts new options via `ReactView#updateOptions` and this will trigger a rerender on the whole React component rendered via particular `ReactView` instance.

rjeczalik committed:
kd: switch to XHR transport for kite

rjeczalik committed:
kd: fix formatting

rjeczalik committed:
kd: add "stack create" subcommand

rjeczalik committed:
kd: rename lazy -> kloud

rjeczalik committed:
kd/kloud: add credential and stack package

rjeczalik committed:
kd: rework credential subcommand

rjeczalik committed:
object: add Printer for generic printing

rjeczalik committed:
kd: rework stack subcommand

rjeczalik committed:
stack/provider: improve template marshaling

rjeczalik committed:
kd: update dependencies versions

rjeczalik committed:
kd: add workaround for vagrant 1.8.7

rjeczalik committed:
klient: fix uploading vagrant logs

anonrig committed:
Add integration test suite

anonrig committed:
Add editorconfig

anonrig committed:
Finalize modal with current components

anonrig committed:
Add TextFooter to modal component

anonrig committed:
Add Sidebar tests

anonrig committed:
Finalize mocha styling

anonrig committed:
Add Rainforest to Mocha test generator

rjeczalik committed:
klient: fix ownership of KODING_HOME

rjeczalik committed:
klient: fix klient.sh

rjeczalik committed:
kd: display identifier first

anonrig committed:
Update test generator

To require required modules in before, and change mocha styling

anonrig committed:
Check test count if a coffee test exists

rjeczalik committed:
kloud/stack: add ImportRequest / ImportResponse structs

anonrig committed:
Fix generator syntax

usirin committed:
ide/test: use `updateOptions` to update modal options

rjeczalik committed:
provider/softlayer: fix for a false-positive verification

Commit:
Rename kodingdev_master key variable

Signed-off-by: Sonmez Kartal <sonmez@koding.com>

rjeczalik committed:
object: remove Printer

rjeczalik committed:
kd/kloud/credential: simplify (*Client).cache()

rjeczalik committed:
provider/softlayer: disable debug logging

anonrig commented:
I've rebased and force pushed to master branch. Can you get rid off the old commits with old commit id? @usirin 

Thanks!

gokmen committed:
Merge pull request #9704 from szkl/kodingdev_master

Rename kodingdev_master key variable

rjeczalik committed:
Merge pull request #9719 from koding/softlayer-quiet

provider/softlayer: disable debug logging

rjeczalik committed:
Merge pull request #9718 from koding/softlayer-verification-fix

provider/softlayer: fix for a false-positive verification

sinan committed:
stack-editor: search box esc binding

sinan committed:
stack-editor: searchbox: placeholder updated

sinan committed:
stack-editor: searchbox: docs link taken to searchbox

sinan committed:
stack-editor: searchbox: minor typo

sinan committed:
stack-editor: searchbox: styles updated

sinan committed:
stack-editor: interim styling

hakankaradis committed:
stack-editor: fix credential list styling

hakankaradis committed:
stack-editor: update stacktitle font color when selected

hakankaradis committed:
stack-editor: read me preview button style fix

hakankaradis committed:
stack-editor: clone of style update

hakankaradis committed:
stack-editor: fix preview buttons style

rjeczalik committed:
kd/kloud: refactor Client with Transport interface

rjeczalik committed:
kd: add unittest runner - MainCmd

rjeczalik committed:
provider/softlayer: use unique SSH keys per bootstrap

Fixes the following tf error:

    * softlayer_ssh_key.koding: Error creating SSH Key: SSH key already exists.

rjeczalik committed:
Merge pull request #9736 from koding/softlayer-ssh-fix

provider/softlayer: use unique SSH keys per bootstrap

rjeczalik committed:
kd: add tests for "credential create"

rjeczalik committed:
provider/marathon: do no try to read Stack on plan request

Fixes nil ptr deref:

  Nov 24 17:22:02 sandbox-54.146.220.126 kloud.log: koding/kites/kloud/provider/marathon.(*Stack).ApplyTemplate(0xc42018c460, 0xc42069ad20, 0x8, 0xc42069ad20, 0x0)
  Nov 24 17:22:02 sandbox-54.146.220.126 kloud.log: /var/app/current/go/src/koding/kites/kloud/provider/marathon/stack.go:112 +0x213

rjeczalik committed:
kd: use KD_EXPERIMENTAL for tests

Commit:
Merge pull request #9741 from koding/marathon-fix

provider/marathon: do no try to read Stack on plan request

Commit:
Merge pull request #9666 from koding/kd-improvements

kd: rework credential / stack commands

gokmen committed:
Merge pull request #9734 from koding/interim-stack-editor

Interim stack editor

usirin commented:
As long as you don't rebase it to the koding/master these commits will always be here.

usirin commented:
~I rebased to latest koding master right now and sent the change for that. If you rebase your master against the original, this pr will be ready. Please don't make me force push to your master, and rebase your own :)~
